src/platform/plugins/shared/dashboard/docs/openapi/entrypoint.yaml

```diff
@@ -0,0 +1,2658 @@
+openapi: 3.0.3
+info:
+  title: Dashboards (unstable)
+  description: |
+    The Dashboards API allows you to manage and interact with Kibana dashboards.
+    You can create, update, delete, and retrieve dashboards programmatically.
+  version: "0.1"
+  contact:
+    name: Kibana Presentation Team
+  license:
+    name: Elastic License 2.0
+    url: https://www.elastic.co/licensing/elastic-license
+tags:
+  - name: dashboards
+    description: Dashboards APIs enable you to manage and interact with Kibana dashboards.
+servers:
+  - url: "https://{kibana_url}"
+    variables:
+      kibana_url:
+        default: localhost:5601
+paths:
+  "/api/dashboards/dashboard":
+    get:
+      description: This functionality is unstable and will be changed or removed in a future release. Unstable features are not subject to the support SLA of official GA features.
```

Comment:
Not sure if we need a legal review of this or not. This is modeled after the technical preview warnings.